app/middleware/notInEnglandHandler.js

```diff
@@ -0,0 +1,25 @@
+const log = require('../lib/logger');
+const renderer = require('./renderer');
+
+function outsideEngland(country) {
+  return !country.includes('England');
+}
+
+function notInEnglandHandler(req, res, next) {
+  const location = res.locals.postcodeLocationDetails;
+  const postcodeSearch = res.locals.postcodeSearch;
+
+  log.debug({ location }, 'notInEnglandHandler');
+
+  if (location) {
+    if (outsideEngland(location.country)) {
```

Comment:
Simplified. 

The else block is required since next and res.render do not return.